tvb_documentation/tutorials/tutorial_1_BuildingYourOwnBrainNetworkModel.rst

```diff
@@ -340,61 +340,60 @@ introduce two major components, that is:
 
 The first significant thing to note about surface simulations is that *Monitors*
-certain make a lot more sense in this context than they do at the region
-level, and so we’ll introduce a couple new *Monitors* here.
+certainly make a lot more sense in this context than they do at the region
+level, and so we’ll introduce a couple of new *Monitors* here.
 
 
 6. The first of these new *Monitors* is called **SpatialAverage**. To select
-   several monitors press the key Command or Control while you select
-   them.
+   several monitors just make sure you check the right boxes.
 
 7. The second of these new monitors, which is an instantiation of a
    biophysical measurement process, is called **EEG**. The third will be
-   the **Temporal Average**.
+   the already known **Temporal Average** monitor.
 
-8. The *Monitors period* is the value **1.953125 ms** which is equivalent to a
+8. The *Sampling period (ms)* for all three monitors is **1.953125 ms** which is equivalent to a
    sampling frequency of 256 Hz.
 
-9. Lastly, the *simulation length* is **500 ms**.
+9. The *simulation length* is **500 ms**.
 
-10. Run the simulation.
+10. Lastly, the simulation name is *AnatomyOfASurfaceSimulation* .
 
-11. Once the simulation is finished, without changing any parameters,
-    click on |branch|.
+11. Run the simulation.
 
-These simulations are *AnatomyOfASurfaceSimulation* and
-*AnatomyOfASurfaceSimulation\_branch1*.
+12. Once the simulation is finished, without changing any parameters,
+    launch a branch from it with the name
+    *AnatomyOfASurfaceSimulation\_branch1*.
 
-The first of these new *Monitors*, will average over the space (nodes) of the
+The first of these new *Monitors* that we mentioned will average over the space (nodes) of the
 simulation. The basic mechanism is general, in the sense that the nodes
 can be broken up into any non-overlapping, complete, set of sets. In
 other words, each node can only be counted in one collection and all
 nodes must be in one collection.
 
 The second of these new Monitors, *EEG*, hopefully also unsurprisingly,
 returns the EEG signals resulting from the simulated neural dynamics
-using in the process a lead-field or *Projection Matrix*.
+using, in the process, a lead-field or *Projection Matrix*.
 
 EEG signals measured on the scalp depend strongly on the location and
 orientation of the underlying neural sources, which is why this monitor
 is more realistic and useful in the case of surface based simulations –
 where the simulation is run on the explicit geometry of the cortex,
 which can potentially have been obtained from a specific individual’s
-brain. In addition a simulation being built on the specific anatomical
+brain. In addition, a simulation being built on the specific anatomical
 structure of an individual subject, the specific electrodes used in
 experimental work can also be incorporated, providing a link between
 simulation and experiment. 
```

Comment:
The following image is changed to project activity on the EEG cap, you can generate the exact initial image if you use Brain Skull surface instead of EEG Cap as projection surface in the TS viewer.
I think it makes more sense to keep it as before as the EEG sensors are usually implanted on the skull.